src/components/protocol_handler/test/protocol_handler_tm_test.cc

```diff
@@ -540,6 +542,176 @@ TEST_F(ProtocolHandlerImplTest, StartSession_Protected_SessionObserverAccept) {
 
   EXPECT_TRUE(waiter->WaitFor(times, kAsyncExpectationsTimeout));
 }
+
+static std::vector<uint8_t> CreateVectorFromBsonObject(const BsonObject* bo) {
+  std::vector<uint8_t> output;
+  if (bo != NULL) {
+    size_t len = bson_object_size(const_cast<BsonObject*>(bo));
+    uint8_t* bytes = bson_object_to_bytes(const_cast<BsonObject*>(bo));
+    output.assign(bytes, bytes + len);
+    free(bytes);
+  }
+  return output;
+}
+
+/*
+ * Simulate two StartService messages of video service from mobile.
+ * Session observer accepts the first message with delay, while rejects the
+ * second message immediately.
+ */
+TEST_F(ProtocolHandlerImplTest,
+       StartSession_Unprotected_Multiple_SessionObserverAcceptAndReject) {
+  using namespace protocol_handler;
+
+  ON_CALL(protocol_handler_settings_mock, enable_protocol_5())
```

Comment:
The `enable_protocol_5` function was deleted in develop in favor of `max_supported_protocol_version`, you're probably best off rebasing on or merging develop.